tests/unittests/config/test_cc_bootcmd.py

```diff
@@ -100,24 +100,6 @@ def test_handler_creates_and_runs_bootcmd_script_with_instance_id(self):
             my_id + " iid-datasource-none\n", util.load_text_file(out_file)
         )
 
-    def test_handler_runs_bootcmd_script_with_error(self):
```

Comment:
Why is this no longer needed? The log is gone but we should still get the exception.